spec/mailers/user_mailer_spec.rb

```diff
@@ -67,36 +106,56 @@
     end
   end
 
-  describe :issue_update do
-    context :delayed_job do
-      before do
-        # Delayed Job does not preserve the closure, so the context of the
-        # delayed method call does not contain the user anymore, who triggered
-        # the job. Instead, the anonymous user is returned.
-        User.current = User.anonymous
+  describe :work_package_added do
+    before do
+      UserMailer.work_package_added(recipient, work_package, user).deliver
+    end
 
-        UserMailer.work_package_updated(user, journal, user)
-      end
+    it_behaves_like 'mail is sent'
```

Comment:
But in a different context. The preconditions differ, that is why I found it sensible to have both.